src/lib/storage/table.cpp

```diff
@@ -203,4 +203,6 @@ TableType Table::get_type() const {
   }
 }
 
+std::vector<std::vector<ColumnID>> Table::get_column_ids_of_indexes() const { return _indexes; }
```

Comment:
This is a weird thing for a table to do. Maybe the index info (column ids, type, later the name) should be a struct returned by `Table::indexes()`?